### ISSUE TITLE
Recent commit broke wal-e

### ISSUE THREAD

**Issue by PSG-Luna** [wal-e/wal-e#234]
Commit c4acf47767daf1c5a633a90fcebc2e0bbaf1cfc9 broke wal-e.

Adding those options in front of the command causes wal-e to fail with the following error:

```
postgres@sql-main:/home/luna$ WALE_GS_PREFIX="gs://sql-main-backup" wal-e --gs-instance-metadata backup-push /mnt/sql-main-data/data/
wal_e.main   INFO     MSG: starting WAL-E
        DETAIL: The subcommand is "backup-push".
        STRUCTURED: time=2016-04-02T12:11:00.804809-00 pid=1305
wal_e.main   ERROR    MSG: could not run one or more external programs WAL-E depends upon
        DETAIL: Could not run the following programs, are they installed? PGOPTIONS="--statement-timeout=0" psql
        STRUCTURED: time=2016-04-02T12:11:00.891744-00 pid=1305
```

When strace'ing to get the syscalls that were run, it's clear what's happening, here's the output:

```
[pid  1844] execve("/usr/local/sbin/PGOPTIONS=\"--statement-timeout=0\" psql", ["PGOPTIONS=\"--statement-timeout=0"..., "-d", "postgres", "--no-password", "--no-psqlrc", "-c", "COPY (SELECT 1) TO STDOUT WITH C"...], [/* 19 vars */]) = -1 ENOENT (No such file or directory)
[pid  1844] execve("/usr/local/bin/PGOPTIONS=\"--statement-timeout=0\" psql", ["PGOPTIONS=\"--statement-timeout=0"..., "-d", "postgres", "--no-password", "--no-psqlrc", "-c", "COPY (SELECT 1) TO STDOUT WITH C"...], [/* 19 vars */]) = -1 ENOENT (No such file or directory)
[pid  1844] execve("/usr/sbin/PGOPTIONS=\"--statement-timeout=0\" psql", ["PGOPTIONS=\"--statement-timeout=0"..., "-d", "postgres", "--no-password", "--no-psqlrc", "-c", "COPY (SELECT 1) TO STDOUT WITH C"...], [/* 19 vars */]) = -1 ENOENT (No such file or directory)
[pid  1844] execve("/usr/bin/PGOPTIONS=\"--statement-timeout=0\" psql", ["PGOPTIONS=\"--statement-timeout=0"..., "-d", "postgres", "--no-password", "--no-psqlrc", "-c", "COPY (SELECT 1) TO STDOUT WITH C"...], [/* 19 vars */]) = -1 ENOENT (No such file or directory)
[pid  1844] execve("/sbin/PGOPTIONS=\"--statement-timeout=0\" psql", ["PGOPTIONS=\"--statement-timeout=0"..., "-d", "postgres", "--no-password", "--no-psqlrc", "-c", "COPY (SELECT 1) TO STDOUT WITH C"...], [/* 19 vars */]) = -1 ENOENT (No such file or directory)
[pid  1844] execve("/bin/PGOPTIONS=\"--statement-timeout=0\" psql", ["PGOPTIONS=\"--statement-timeout=0"..., "-d", "postgres", "--no-password", "--no-psqlrc", "-c", "COPY (SELECT 1) TO STDOUT WITH C"...], [/* 19 vars */]) = -1 ENOENT (No such file or directory)
```

I think the commit should be reverted

**Comment by vmihailenco** [comment]
Fixed by https://github.com/wal-e/wal-e/pull/233, but needs merging

**Comment by fdr** [comment]
I merged #233, thanks for being involved, folks.
